src/main/play-doc/operation/Install.md

```diff
@@ -531,6 +533,40 @@ A typical response contains the current members of the cluster (shown here is a
 
 ```
 
+### Loading and Running ConductR-HAProxy Bundle
+
+ConductR-HAProxy bundle listens for bundle changes within ConductR and updates the HAProxy config to expose the bundle endpoints accordingly.
+
+#### Prepare ConductR-HAProxy nodes
+
+_Repeat each step in this section on each node._
+
+ConductR-HAProxy bundle must be installed on all nodes where HAProxy is installed, and these nodes can be distinguished by the `haproxy` role. Assign the `haproxy` role to the nodes where the proxy will be hosted.
+
+Append the `haproxy` role to the default `web` role as follows:
+
+```bash
+echo -Dakka.cluster.roles.0=web | sudo tee -a /usr/share/conductr/conf/application.ini
+echo -Dakka.cluster.roles.1=haproxy | sudo tee -a /usr/share/conductr/conf/application.ini
+sudo service conductr restart
+```
+
+### Use CLI to load and run ConductR-HAProxy bundle
+
+_Execute the step in only on the ConductR seed node._
+
+Loading and running the ConductR-HAProxy bundle requires [[CLI|CLI]] to be installed. Continue with the next step once [[CLI|CLI]] is installed.
+
+Load and run the ConductR-HAProxy bundle as follows. Scale ConductR-HAProxy so that ConductR-HAProxy is running on every proxy node in the cluster. In our case we have 3 nodes where the proxy is expected to be running, so we scale up the ConductR-HAProxy to 3 instances.
```

Comment:
@edwardcallahan - Added 'Scale ConductR-HAProxy so that ConductR-HAProxy is running on every proxy node in the cluster' before 'In our case...'